srcpkgs/dracut/files/kernel-uefi-hook-postinst

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# Kernel post-install hook for dracut EFI bundle.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+. "${ROOTDIR}/etc/default/dracut-uefi-hook"
+if [ "x${CREATE_UEFI_BUNDLES}" != x1 ]; then
+	exit 0
+fi
+
+if [ ! -x bin/dracut ]; then
+	exit 0
+	fi
+
+if [ -z "${KERNEL_CMDLINE}" ] && [ "x${SKIP_KERNEL_CMDLINE}" != x1 ]; then
+	echo "ERROR: You need to set the kernel command line in /etc/default/dracut-uefi-hook"
+	exit 1
+fi
+
+OPTIONS="${OPTIONS} --force --uefi ${UEFI_BUNDLE_DIR:=boot/efi/EFI/Void}/linux-${VERSION}.efi ${VERSION}"
+
+dracut --kernel-cmdline="${KERNEL_CMDLINE}" ${OPTIONS}
```

Comment:
`dracut` is talkative, `-q` should be used, I think.

Reply:
Yup!